client/src/components/ModalTemplate/SampleLoginForm.js

```diff
@@ -1,7 +1,9 @@
 import React, { useState } from 'react';
 import { StyledModal } from './StyledModal';
+import {initializeApp} from "firebase/app";
 
-const SampleLoginForm = () => {
+
+const LoginForm = () => {
```

Comment:
line 37 `export deafault SampleLoginForm` does not match defined `const LoginForm = () =>{}` on line 6

Needs to be updated before merging

Reply:
Changed as requested.